services/editor/src/pages/settings/components/PoliciesPage/utils.ts

```diff
@@ -0,0 +1,64 @@
+
+import * as R from "ramda";
+import {useState, useEffect, useContext} from 'react';
+import { ReduxContext } from "../../../../store";
+import { showError } from "../../../../store/ducks/notifications";
+import { FetchError } from "tweek-client";
+
+type LoadingState = "idle" | "saving" |"loading" | "error";
```

Comment:
const enum? :grimacing:

Reply:
I started with const enum, I think there was a problem with babel integration with TS.
I can try again.

Reply:
I'm getting:
SyntaxError: /Users/yshayy/code/tweek/services/editor/src/pages/settings/components/PoliciesPage/utils.ts: 'const' enums are not supported.

Reply:
I think I investigated it last week, and it was related to create-react-app compilation pipeline of TS through babel or something like that...

Reply:
Also, build is ridiculously slow after TS, need to investigate.
(seems to be work on it in https://github.com/facebook/create-react-app/pull/5903, although the main slow I see is in making production build, not dev)